pkg/meta/dynamodb/dynamodb_test.go

```diff
@@ -151,6 +151,11 @@ func TestWrapperErrors(t *testing.T) {
 
 	log := log.NewLogger("debug", "")
 
+	imgTrustStore, err := imagetrust.NewAWSImageTrustStore(region, endpoint)
+	if err != nil {
+		panic(err)
```

Comment:
We have a logger here right? Can we log.Fatal() instead?

Reply:
This is a test, it needs to exit if it cannot create the object.

Reply:
Switch to using the test framework specific methods.